pwa-devdocs/src/tutorials/pwa-studio-fundamentals/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: PWA Studio fundamentals
+---
+
+## Overview
+
+This tutorial series guides you through setting up, working with, and deploying a [Magento PWA Studio][] storefront.
+
+These tutorials teach you how to use PWA Studio by providing recommended practices for storefront development.
+During the course of the tutorials, you will create a new storefront application and work with the tools provided by PWA Studio to make basic changes to the initial site.
+
+![what we are building][]
+
+## Prerequisites
+
+-   A basic understanding of [React][]
+-   Node >= 10.14.1
+-   Yarn (recommended) or NPM
+
+## Tutorials available
+
+-   [Project setup][] - Setup a local development environment using the scaffolding tool
+-   Project Structure & Routing - Understand the project structure and learn the basics of how routing works in PWA Studio.
```

Comment:
Are these eventually going to be linked?

Reply:
Yes